recipe/meta.yaml

```
@@ -13,9 +13,8 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 2
-  # arrow-cpp is not available for s390x, see arrow-cpp-feedstock for details.
-  skip: true  # [s390x or py<37]
+  number: 0
+  skip: true  # [py<37]
```

```suggestion
  skip: true  # [py<38]
```
https://github.com/apache/arrow/blob/740889f413af9b1ae1d81eb1e5a4a9fb4ce9cf97/python/setup.py#L498